packages/gasket-plugin-command/README.md

```diff
@@ -0,0 +1,201 @@
+# @gasket/plugin-command
+
+Enable plugins to inject new commands to be available for use with the Gasket
+CLI. Executes the `commands` lifecycle in the `configure` hook of the
+`@gasket/plugin-command` plugin. This plugin utilizes [Commander.js] and additional documentation can be found in the documentation.
+
+## Installation
+
+```bash
+npm i @gasket/plugin-command
+```
+
+Update your `gasket.js` plugin configuration:
```

Comment:
Is this something developers will typically need to add to their app? I'm guessing it'll usually be included in presets, so maybe a note would be handy for that. Overall, I see an anti-pattern for Gasket users where they're frequently redundantly installing plugins.

Reply:
It will typically be adding during create-gasket-app with a preset.

Reply:
This is completely optional plugin and will NOT be included in any presets which is the opposite of version 6. Due to the foundational changes of version 7, apps will not be reliant on the Gasket CLI and or executing scripts with the CLI, which leads to this plugin being completely optional and intended to be used if the user needs a custom command option.